store/src/java/com/zimbra/cs/service/FileUploadServlet.java

```diff
@@ -718,12 +717,11 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         List<FileItem> items = new ArrayList<FileItem>(1);
         items.add(fi);
         Upload up = new Upload(acct.getId(), fi, filename);
-        
+
         if (filename.endsWith(".har")) {
             File file = ((DiskFileItem) fi).getStoreLocation();
             try {
-                Tika tika = new Tika();
-                String mimeType = tika.detect(file);
+                String mimeType = MimeDetect.getMimeDetect().detect(file);
```

Comment:
We would also need to verify ZBUG-711 with this change

Reply:
Yes I have and added it to testing notes